runtime/doc/lsp.txt

```diff
@@ -1840,6 +1840,9 @@ get_lines({uri}, {rows})                            *vim.lsp.util.get_lines()*
                 Return: ~
                     table<number string> a table mapping rows to lines
 
+get_markdown_fences()                     *vim.lsp.util.get_markdown_fences()*
```

Comment:
@folke probably should mark this private

Reply:
Oops, you're right. I'll change it in a bit